includes/packages/Manuel-Shipment/manual-shipment.php

```diff
@@ -0,0 +1,297 @@
+	    $order_id=$_GET['post'];
+	    $hez_ship_track_no= get_post_meta( $order_id, 'hez_ship_track_no', true );
+	    $hez_ship_carrier= get_post_meta( $order_id, 'hez_ship_carrier', true );
+		?>
+		<style type="text/css">
+			.shipping_fields{
+				width: 100%;
+			}
+			.update_shipping{
+				float: right;
+			}
+			.wp-clearfix{
+				clear: both;
+				margin-top: 5px;
+			}
+			.text-right{
+				text-align: right;
+			}
+			.error_input{
+				border: 1px solid #d63638 !important;
+			}
+		</style>
+		<form action="" method="post" >
+		    <div class="wp-clearfix">
+			    <label>Shipping Carrier</label><br>
+			    <select name="hez_ship_carrier" class="shipping_fields hez_ship_carrier">
+			    	<option value="">Select Shippiing Carrier</option>
+			    <?php 
+			    foreach($shipping_carriers as $key=>$shipping_carrier)
+			    {
+			    	if($shipping_carrier['is_available'])
+			    		echo "<option value='".$key."' ".($hez_ship_carrier==$key?'selected':'').">".$shipping_carrier['name']."</option>";
+			    }
+			    ?>
+			    </select>
+			</div>
+			
+		    <div class="wp-clearfix">
+		    	<label>Shipping Tracking Code</label><br>
+		    	<input type="text" name="hez_ship_track_no"  value="<?php echo $hez_ship_track_no; ?>" class="shipping_fields hez_ship_track_no">	
+		    </div>
+		    <br>
+		    <div class="text-right">
+		    	<button type="button" name="update_shipping" class="button button-primary update-shipping">Update Shipping</button>	
+		    </div>
+		    
+		</form>
+		<script>
+	        jQuery(document).ready(function($) {
+	            var order_id=<?php echo $order_id; ?>;
+	            $('.update-shipping').on('click',function(){
+	                $('.loader').fadeIn();
+	                $('.update-shipping').attr('disabled','disabled');
+	                var error=0;
+	                var hez_ship_carrier=$('.hez_ship_carrier').val();
+	                var hez_ship_track_no=$('.hez_ship_track_no').val();
+	                if(hez_ship_carrier==''){
+	                	error++;
+	                	$('.hez_ship_carrier').addClass('error_input');
+	                }
+	                if(hez_ship_track_no==''){
+	                	error++;
+	                	$('.hez_ship_track_no').addClass('error_input');
+	                }
+	                if(error==0){
+		                $.ajax({
+		                     type : "POST",
+		                     dataType : "json",
+		                     url : "<?php echo home_url();?>/wp-admin/admin-ajax.php",
+		                     data : {action: "hezarfen_shipping_data_action",id:order_id,hez_ship_carrier:hez_ship_carrier,hez_ship_track_no:hez_ship_track_no},
+		                     success: function(response) {
+		                           $('.update-shipping').removeAttr('disabled');
+		                           $('.hez_ship_track_no').removeClass('error');
+		                           $('.hez_ship_carrier').removeClass('error');
+		                           location.reload();
+
```

Comment:
we don't need location.reload since we've AJAX support. We should avoid hard reload